packages/desktop-client/src/icons/v1/ChartAnalytics.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import type { SVGProps } from 'react';
+export const SvgChartAnalytics = (props: SVGProps<SVGSVGElement>) => (
```

Comment:
The AreaGraph is a line graph icon. Is there a better one we can use for AreaGraph so that the line graph can look more unique?

Reply:
It looks like we're already using all three graph-related icons from zondicons. I looked for a line chart one that kinda sorta fit into the style, to have something different from ionicons, but neither have great matches for an area chart. Finding something in a third set will definitely look...not clean. Unless we want to move all of the icons on this page to another icon set?

Reply:
Use the SVGChart for the line graph. It properly matches the graphType.

I've added a new Area Graph icon (#2526) to better match that graph. I always intended to do this but knew I wouldn't NEED to until I integrated the line graph. So here we are...